.travis.yml

```diff
@@ -26,98 +26,100 @@ branches:
   only:
     - master
 
+# TODO: Test on the minimum and maximum supported platform versions
+
 jobs:
   include:
     - stage: Node tests
-      osx_image: xcode10.2
+      osx_image: xcode10
       language: node_js
-      node_js: "10"
+      node_js: "12"
```

Comment:
Why bump this? 10 is still supported.

Reply:
to change less in the future:)

Reply:
v12 is also LTS

Reply:
But Appium still has to support 10. Given that it changes once a year, seems like a small change to have to do.

Reply:
Bear in mind you can build using multiple versions of node ;)
Also, v12 will be LTS, for now it's current